mlflow/tracking/client.py

```diff
@@ -396,6 +396,39 @@ def print_experiment_info(experiments):
             view_type=view_type, max_results=max_results, page_token=page_token
         )
 
+    def search_experiments(
+        self,
+        view_type: int = ViewType.ACTIVE_ONLY,
+        max_results: Optional[int] = None,
+        filter_string: Optional[str] = None,
+        order_by: Optional[List[str]] = None,
+        page_token=None,
+    ) -> PagedList[Experiment]:
+        """
+        Search for experiments that match the specified search criteria.
+
+        :param view_type: Qualify requested type of experiments.
+        :param max_results: If passed, specifies the maximum number of experiments desired. If not
+                            passed, all experiments will be returned for the File and SQL backends.
+                            For the REST backend, the server will pick a maximum number of results
+                            to return.
+        :param filter_string: Filter query string, defaults to searching for all experiments.
```

Comment:
Can we add some examples of valid `filter_strings`? Can we also add these to the fluent API docs?

Reply:
Got it, I'll some examples and also describe supported attributes and operators.